CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
 ## 2.12.0 (Unreleased)
+
+FEATURES:
+* **New Resource:** `gitlab_service_pipeline_emails`
```

Comment:
Usually the maintainer takes care of the changelog, but if you're going to update it yourself, be sure to use the correct format to reference the issue. Example: https://github.com/terraform-providers/terraform-provider-gitlab/commit/f05eed843ae5a6e1725f9e2523e6cf849371d846

Reply:
I copied from:
```
FEATURES:
* **New Resource:** `gitlab_service_github`
  ([#311](https://github.com/terraform-providers/terraform-provider-gitlab/issues/311))
```
in the diff you linked. I'd be happy to fix my changelog, but I don't really what differs from usual?

Reply:
The tf-release-bot expects the form `[GH-375]`, where 375 is the PR or issue number. During a release it replaces those references with the markdown format you saw.

https://www.terraform.io/docs/extend/best-practices/versioning.html#entry-format

Bot commit be like https://github.com/terraform-providers/terraform-provider-gitlab/commit/d48f13a9bfb54a161b2bd25fa7de92340f65f3ea

Reply:
OK sure, I'll change what I did so that a bot can do the exact same thing later :D 
Thank you very much for the kid review. I was a little light on reading the documentation.

Reply:
I'll just remove my changelog changes as recommended by Hashicorp. BTW, the Changelog of this Repo does no follow the specification!